src/forms/StudyFormSteps.js

```diff
@@ -259,56 +318,121 @@ export const GrantStep = ({
+            <Button
+              attached="bottom"
+              onClick={() => setFoldDescription(!foldDescription)}
+              className="mb-15"
+            >
+              {foldDescription ? 'Read More' : 'Read Less'}
+            </Button>
+          )}
+        </>
+      ) : (
+        <FormField
+          newStudy={newStudy}
+          id="description"
+          name="Description"
+          description="Study description in markdown, commonly the X01 abstract
       text."
-        type="text"
-        focused={focused === 'description'}
-        value={values.description}
-        touched={touched.description}
-        errors={errors.description}
-        handleChange={handleChange}
-        handleBlur={handleBlur}
-        handleFocus={id => setFocused(id)}
-      >
-        <Form.TextArea
-          rows="15"
           type="text"
-          name="description"
-          onChange={handleChange}
-          onBlur={handleBlur}
+          focused={focused === 'description'}
           value={values.description}
-        />
-      </FormField>
+          touched={touched.description}
+          errors={errors.description}
+          handleChange={handleChange}
+          handleBlur={handleBlur}
+          handleFocus={id => setFocused(id)}
+        >
+          <Form.TextArea
+            className="noMargin"
+            rows="15"
+            type="text"
+            name="description"
+            onChange={handleChange}
+            onBlur={handleBlur}
+            value={values.description}
+            readOnly={!editing && !newStudy}
+          />
+        </FormField>
+      )}
       <Button
         primary
         floated="left"
         type="button"
-        onClick={() => history.push('/study/new-study/external')}
+        onClick={() => {
+          if (newStudy) {
+            history.push('/study/new-study/external');
+          } else {
+            history.push(
+              '/study/' +
+                history.location.pathname.split('/')[2] +
+                '/basic-info/external',
+            );
+          }
+        }}
         labelPosition="left"
         icon="left arrow"
         content="PREVIOUS"
       />
-      <Button
-        primary
-        floated="right"
-        type="button"
-        disabled={
-          Object.keys(errors).length > 0 ||
-          values.name.length === 0 ||
-          values.externalId.length === 0
-        }
-        onClick={() => {
-          validateForm().then(errors => {
-            Object.keys(errors).length === 0 && setConfirmOpen(true);
-          });
-        }}
-      >
-        SUBMIT
-      </Button>
+      {newStudy && (
+        <Button
+          primary
+          floated="right"
+          type="button"
+          disabled={
+            Object.keys(errors).length > 0 ||
+            values.name.length === 0 ||
+            values.externalId.length === 0
+          }
+          onClick={() => {
+            validateForm().then(errors => {
```

Comment:
Because new study view and study info view are calling the same `NewStudyForm` component. `newStudy` is a prop used to conditionally render different field and look for creating new study or showing/editing existing study info.

Here is how the form is called from new study view with `newStudy` prop set to `true`
```
<NewStudyForm
    newStudy
    submitValue={submitValue}
    apiErrors={newStudyError}
    history={history}
/>
```

Reply:
Formik has validation on submit, but here we want to trigger the validation before submitting. This button supposes to open the confirm modal.